lib/xwax/timecoder.c

```diff
@@ -430,7 +431,12 @@ void timecoder_init(struct timecoder *tc, struct timecode_def *def,
     tc->dt = 1.0 / sample_rate;
     tc->sample_rate = sample_rate;
     tc->zero_alpha = tc->dt / (ZERO_RC + tc->dt);
-    tc->threshold = ZERO_THRESHOLD;
+
+    if (tc->def->flags & TRAKTOR_MK2)
+        tc->threshold = ZERO_THRESHOLD_MK2;
+    else
+        tc->threshold = ZERO_THRESHOLD;
+
```

Comment:
NIT: consider moving to `timecode_def` instead of adding yet more `tc->def->flags & TRAKTOR_MK2` (IMO) hacks?